pkg/testcommon/testcommon_test.go

```diff
@@ -57,15 +57,15 @@ func TestChdir(t *testing.T) {
 	CleanupDir(testDir)
 }
 
-// TestCaptureStdOut ensures capturing of standard out works as expected.
+// TestCaptureStdOut ensures capturing of UserOut works as expected.
```

Comment:
this is still capture of stdout though no?

Reply:
Fixed comment, added minor improvement to capture in the test itself.